File: BoJ/python/level2(if문)/알람 시계.py
a, b = input().split()
a = int(a)
b = int(b)

if b < 45:
    if a > 0:
        print(a-1, b-45+60)
    else:
        print(23, b-45+60)
else:
    print(a, b-45)

File: BoJ/Java/단계별로 풀어보기/level2(if문)/src/알람_시계.java
import java.io.*;

public class 알람_시계 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        String[] input = br.readLine().split(" ");
        int h = Integer.parseInt(input[0]);
        int m = Integer.parseInt(input[1]);

        int result = h * 60 + m - 45;
        if (result < 0) {
            result = 24 * 60 + result;
        }

        bw.write(result / 60 + " " + result % 60 + "\n");
        bw.flush();
        bw.close();
    }   
}
